examples/26_kimi_k2_5_multimodal.py

```
@@ -0,0 +1,196 @@
+    print(f"  URL:     {deployment.url}")
+    print(f"  State:   {deployment.state}")
+    print()
+    print("API endpoints:")
+    print(f"  Chat:    {deployment.url}/v1/chat/completions")
+    print(f"  Models:  {deployment.url}/v1/models")
+    print(f"  Health:  {deployment.url}/health")
+    print()
+    print("Example - Thinking mode (default):")
+    print(f"  curl {deployment.url}/v1/chat/completions \\")
+    print('    -H "Content-Type: application/json" \\')
+    print(f"    -d '{{")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)